includes/class-wp-session-utils.php

```diff
@@ -100,7 +100,14 @@ public static function delete_old_sessions( $limit = 1000 ) {
 		// Delete expired sessions
 		if ( ! empty( $expired ) ) {
 			$names = implode( "','", $expired );
-			$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name IN ('{$names}')" );
+			$wpdb->query( $wpdb->prepare(
```

Comment:
The problem with using `::prepare()` here is what it will do with the quotes. For example, let's say `$expired` is an array consisting of `['one', 'two', 'three']`. This will mean `$names` consists of, literally, `one', 'two', 'three` - note the position of the single quotes there. Passing this string through `::prepare()` will turn it into: `one\',\'two\',\'three`. The quotes are escaped, and this will result in a broken SQL statement.

Instead, perhaps passing the `$expired` array through `esc_sql()` would be a safer route.

```php
$expired = array_map( 'esc_sql', $expired );
$names = implode( "','", $expired );
// ... old sql statement as-is
```